test/OpenSpaceToolkit/Astrodynamics/Trajectory/Propagator.test.cpp

```diff
@@ -558,19 +563,6 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Trajectory_Orbit_Model_Propagator, AddMane
         EXPECT_EQ(defaultPropagatorWithManeuvers_.getNumberOfCoordinates(), 7);
         EXPECT_EQ(defaultPropagatorWithManeuvers_.getDynamics().getSize(), 4);
     }
-
-    // Create another maneuver with same accelerations but with overlapping instants
-    Array<Instant> overlappingManeuverInstants = Array<Instant>::Empty();
-    for (const Instant& instant : defaultManeuverInstants_)
-    {
-        overlappingManeuverInstants.add(
-            instant + defaultManeuver_.getInterval().getDuration() - Duration::Seconds(1.0)
-        );
-    }
-
-    const Maneuver overlappingManeuver = {
-        overlappingManeuverInstants, defaultManeuverAccelerationProfile_, gcrfSPtr_, defaultManeuverMassFlowRateProfile_
-    };
```

Comment:
I guess this unit test was never actually finished, but it would be good to test that the propagator raises an error here (if its not done anywhere else)

Reply:
Yeah I just removed the incomplete test, feel free to add it back in since I believe you had worked on this test.